packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -12,10 +12,7 @@ export const containerClass = style([
     height: '$16',
     justifyContent: 'flex-start',
     position: 'relative',
-    width: {
-      xs: 'max-content',
-      sm: '100%',
-    },
+    minWidth: 'max-content',
   }),
```

Comment:
I don't think this results in the behavior you would like. Only the links should be scrollable if I understand correctly? @isaozler

Reply:
![image](https://github.com/kadena-community/kadena.js/assets/1378630/ca0ae4cc-6cdf-4b9d-bf62-17152b7adf03)

with minimum space between sections (module links, social icons, and network select) should be scrollable

the logo icon should be fixed